spring-cloud-kubernetes-config/src/main/java/org/springframework/cloud/kubernetes/config/reload/ConfigReloadAutoConfiguration.java

```diff
@@ -48,7 +48,8 @@
  * @author Nicolla Ferraro
  */
 @Configuration(proxyBeanMethods = false)
-@ConditionalOnProperty(value = "spring.cloud.kubernetes.enabled", matchIfMissing = true)
+@ConditionalOnProperty(name = "spring.cloud.kubernetes.config.enabled",
+		havingValue = "true")
```

Comment:
Why this change?

Reply:
change the property itself and explicitly specify it should be true in value for clarity. don't mind using the old pattern.